### PR TITLE
Renson integration

### PR DESCRIPTION
**Pull request by jimmyd-be**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/56374
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2826
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by thecode** [comment]
Hi @jimmyd-be Please rebase your PR to next or close this PR and make a new one, Thanks

File: source/_integrations/renson_endura_delta.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Renson Endura Delta
+description: Instructions on how to integrate Integration Sensor into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 0.87
```
Comment (balloob) [inline source/_integrations/renson_endura_delta.markdown]: not correct

File: source/_integrations/renson_endura_delta.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: Renson Endura Delta
+description: Instructions on how to integrate Integration Sensor into Home Assistant.
```
Comment (balloob) [inline source/_integrations/renson_endura_delta.markdown]: not correct

File: source/_integrations/renson_endura_delta.markdown [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ha_category:
+  - Sensor
+ha_release: 0.87
+ha_iot_class: Local Poll
+ha_quality_scale: internal
```
Comment (balloob) [inline source/_integrations/renson_endura_delta.markdown]: not correct

File: source/_integrations/renson.markdown [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+ha_config_flow: true
+ha_codeowners:
+  - '@jimmyd-be'
+ha_domain: integration
+ha_platforms:
+  - sensor
+  - Binary Sensor
+---
+
+The `integration` platform provides the data from the Renson Endura delta application for Android en iOS. Also the integration provides some services to control the ventilation unit.
```
Comment (frenck) [inline source/_integrations/renson.markdown]:
```suggestion
The Reason integration pulls in data from the Renson Endura delta application for Android en iOS. It also provides some services to control the ventilation unit.
```

File: source/_integrations/renson.markdown [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+## Configuration
+
+### Using UI
+
+The integration can be configured using the UI and this is the required method to configure the integration.
```
Comment (frenck) [inline source/_integrations/renson.markdown]:
```suggestion
{% include includes/config_flow.md %}
```

File: source/_integrations/renson.markdown [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  - platform: renson
+    host: HOST
+```
+
+{% configuration %}
+host:
+  description: The hostname or IP address of the Renson endura delta.
+  required: true
+  type: string
+{% endconfiguration %}
```
Comment (frenck) [inline source/_integrations/renson.markdown]: Adding manual configuration isn't allowed anymore for integrations connecting to a device or service. This part can thus be removed.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/renson.markdown [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+ha_release: 2021.10
+ha_iot_class: Local Poll
+ha_config_flow: true
+ha_codeowners:
+  - '@jimmyd-be'
+ha_domain: integration
+ha_platforms:
+  - sensor
+  - Binary Sensor
+  - Fan
```
Comment (frenck) [inline source/_integrations/renson.markdown]: Platforms for the initial PRs (including those in Core itself) should be limited to a single platform.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/renson.markdown (hunk 7 above)
Comment (jimmyd-be) [reply source/_integrations/renson.markdown]: Removed all platforms except sensor platform.

File: source/_integrations/renson.markdown [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+ha_codeowners:
+  - '@jimmyd-be'
+ha_domain: integration
+ha_platforms:
+  - sensor
+---
+
+The Reason integration pulls in data from the Renson Endura delta application for Android en iOS.
+
+{% include includes/config_flow.md %}
```
Comment (klaasnicolaas) [inline source/_integrations/renson.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/renson.markdown [hunk 9]
```diff
@@ -0,0 +1,19 @@
+---
+title: Renson
+description: Instructions on how to integrate Renson Endura Delta sensors into Home Assistant.
+ha_category:
+  - Sensor
+  - Binary Sensor
```
Comment (MartinHjelmare) [inline source/_integrations/renson.markdown]:
```suggestion
```

File: source/_integrations/renson.markdown [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+ha_release: 2021.10
```
Comment (MartinHjelmare) [inline source/_integrations/renson.markdown]:
```suggestion
ha_release: 2023.7
```

File: source/_integrations/renson.markdown [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+ha_iot_class: Local Poll
+ha_config_flow: true
+ha_codeowners:
+  - '@jimmyd-be'
+ha_domain: integration
```
Comment (MartinHjelmare) [inline source/_integrations/renson.markdown]:
```suggestion
ha_domain: renson
```

File: source/_integrations/renson.markdown [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+ha_platforms:
+  - sensor
+---
+
+The Reason integration pulls in data from the Renson Endura delta application for Android en iOS.
```
Comment (MartinHjelmare) [inline source/_integrations/renson.markdown]: This sounds like the integration interfaces with the app on the phone, but that's not what it does, right?

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/renson.markdown (hunk 12 above)
Comment (jimmyd-be) [reply source/_integrations/renson.markdown]: Changed the text a bit.

**Review by MartinHjelmare — DISMISSED** [review]
Thanks!

**Comment by MartinHjelmare** [comment]
Please address the lint errors too.

**Comment by jimmyd-be** [comment]
> Please address the lint errors too.

@MartinHjelmare Fixed the Lint error

**Review by MartinHjelmare — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**